src/main/java/calculator/input/Symbol.java

```diff
@@ -0,0 +1,38 @@
+package calculator.input;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Symbol {
+    private static final String add = "+";
+    private static final String sub = "-";
+    private static final String mul = "*";
+    private static final String div = "/";
```

Comment:
선언은 상수로되있지만 컨벤션은 변수 컨벤션이네요 🤔 

그리고 이 Symbol은 해당 미션에서는 이 4가지의 사칙연산으로 고정될 것 같은데, 그럼 객체를 미리 만들어 캐싱해도 되지 않을까요? 

```suggestion
    private static final Map<String, Symbol> CACHE_MAP = map.of("+", new Symbol("+"), ...);
```